app/modules/individuals/resources.py

```diff
@@ -515,3 +531,106 @@ def get(self, individual):
             except HoustonException as ex:
                 abort(ex.status_code, ex.message)
             return send_file(image_path, attachment_filename='individual_image.jpg')
+
+
+# TESTING ONLY
+@api.route('/merge_request/test')
+class IndividualMergeRequestCreate(Resource):
+    def get(self):
+        from app.modules.encounters.models import Encounter
+        from flask_login import current_user
+
+        individual = Individual()
+        enc = Encounter()
+        enc.owner = current_user
+        individual.add_encounter(enc)
+        findiv = Individual()
+        with db.session.begin():
+            db.session.add(individual)
+            db.session.add(enc)
+            db.session.add(findiv)
+        params = {'deadline_delta_seconds': 60}  # speed it up
+        res = individual.merge_request_from([findiv], params)
+        log.warning(f'TEST CREATE {individual}, {res}')
+        return {'id': res['async'].id, 'job': str(res), 'individual': str(individual)}
+
+
+@api.route('/merge_request/<uuid:task_id>', defaults={'vote': None}, doc=False)
+@api.route('/merge_request/<uuid:task_id>/<string:vote>')
+class IndividualMergeRequestByTaskId(Resource):
+    """
+    Details of merge request, or register a vote on it
+    """
+
+    def get(self, task_id, vote):
```

Comment:
Why is the GET API being used to POST a vote?
Surely the GET should return the status of the Merge request and POST should be used to vote on it

Reply:
made **vote** now a POST